CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Environment variables user guide
 - Utility for estimating memory requirements of discrete product search space
 - `mypy` for search space and objectives
+- Add `register_callable` to give users the possibility to define their callables
```

Comment:
@Scienfitz the coretest is currently failing due to the same error that was present before merging 0.9.0 (I think), how did you resolve that in the end?

Reply:
no longer relevant since the switch to Python 3.10